src/spotbugs/excludesFilter.xml

```diff
@@ -138,4 +138,10 @@
   <Match>
     <Bug pattern="RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT" />
   </Match>
+  <Match>
+    <Bug pattern="NP_NULL_ON_SOME_PATH" />
+  </Match>
+  <Match>
+    <Bug pattern="NP_NONNULL_RETURN_VIOLATION" />
+  </Match>
```

Comment:
Why are these suppressions being added? They look like legitimate warnings to me.

Reply:
I think there were some problems with some modules before, but maybe we should target specific modules if we really do want to add these exlusions. @harsh-ps-2003 Would you like to follow up?

Reply:
I don't think these suppressions are required. I don't see any errors when doing `mvn verify` without them.

Reply:
Yes, I can confirm now that these changes are no longer required and have been reverted.